docs/api_v4.md

```diff
@@ -46,6 +46,11 @@ If a `historic=1` param is included in the request, one additional field will be
 |-------------:|:---------------------------------------------|:------------------------------------------------------|:----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | `history`    | array of numbers                             | never                                                 | Ordered durations of all previous runs. The first item is the first run recorded by the runner's timer into the source file. The last item is the most recent one. This field is only nonempty if the source timer records history.           |
 
+If an `Accept` header is present, splits.io will try to render the run file natively not in JSON of the type specified.  The format for the header is `application/timer`, with timer being any of the timers that can be exported by splits.io.
```

Comment:
splits.io is the domain, Splits I/O is the website, so usually use the latter when talking about it in a sentence. (splits-io or splitsio is the code base, still undecided on that one but not relevant here)